__tests__/api/__utils__/seeds/users_and_follow_seeds.js

```diff
@@ -102,7 +108,8 @@ const generateUserAndFollowSeeds = async () => {
   return {
     users: bulks,
     followers,
-    followings
+    followings,
+    anonymousUsername
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The returned object from `generateUserAndFollowSeeds` now includes `anonymousUsername`. Ensure that all functions that use the return value of this function are updated to handle this new property.

```diff
+    anonymousUsername
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->